### PR TITLE
Add more logging

### PR DESCRIPTION
**Pull request by eruizgar91**
## Description
Increase logging of squid-py.
## Is this PR related with an open issue?

Related to Issue #173 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

#### Funny gif

Put a link of a funny gif inside the parenthesis-->

### CODE REVIEW

File: squid_py/ocean/account.py [hunk 1]
```diff
@@ -25,7 +27,9 @@ def unlock(self):
         :return: Result of the operation, bool
         """
         if self.password:
+            logger.debug('Unlocking account %s' % self.address)
             return self.keeper.web3.personal.unlockAccount(self.address, self.password)
+        logging.warning('It could not be unlocked the account %s' % self.address)
```
Comment (ttmc) [inline squid_py/ocean/account.py]:
```suggestion
        logging.warning('Failed to unlock the account %s' % self.address)
```

File: squid_py/ocean/ocean.py [hunk 2]
```diff
@@ -258,7 +262,7 @@ def register_asset(self, metadata, publisher, service_descriptors):
             url=ddo_service_endpoint,
             account=publisher
         )
-
+        logger.info('DDO with did %s successfully registered on chaing.' %did)
```
Comment (ttmc) [inline squid_py/ocean/ocean.py]:
```suggestion
        logger.info('DDO with did %s successfully registered on chain.' %did)
```

File: tests/test_secret_store.py [hunk 3]
```diff
@@ -13,6 +13,7 @@
 publisher_password = 'password'
 
 
+# TODO This test is passing running docker-images. We need to add it again when we run IT in the flow.
```
Comment (jcortejoso) [inline tests/test_secret_store.py]: There are some project running these kind of tests using barge. Maybe we can open an issue about this? I can take a look to that.

Comment (eruizgar91) [reply tests/test_secret_store.py]: That would be great :)

**Pull request merged**